### PR TITLE
Шутина - Лабораторные работы #0, #1

### PR DESCRIPTION
**Pull request by ShutinaSvetlana**

### CODE REVIEW

**Comment by ShutinaSvetlana** [comment]
@GerasimovDG

**Comment by SylaiseElvenan** [comment]
👍

**Comment by GerasimovDG** [comment]
👍

File: code/ShutinaSvetlana/batcher.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import math
+
+
+class Sorting:
+    # class realizes the sorting for array of numbers
+    arr = []
+
+    def __init__(self, arr):
+        self.arr = arr
```
Comment (sawirani) [inline code/ShutinaSvetlana/batcher.py:12]: What will happen if we will not give the correct data? Please add tests for this.

**Review by sawirani — CHANGES_REQUESTED** [review]

**Review by sawirani — APPROVED** [review]

**Pull request merged**